TLM/TLM/Util/Shortcuts.cs

```diff
@@ -203,5 +206,18 @@ internal static class Shortcuts {
         internal static ushort GetFarSegment(this ref NetSegment segment, ushort nodeId) =>
             LHT ? segment.GetRightSegment(nodeId) : segment.GetLeftSegment(nodeId);
         #endregion
+
+        internal static Bezier3 ForceHeight(this Bezier3 bezier, float height) {
```

Comment:
won't this create a copy?
more refs plz :D

Reply:
I expect copy there because I don't want to modify original netInfo lane 😉 I'll add comment